test/test_content/songs.control

```diff
@@ -0,0 +1 @@
+["exsong.sg", "texsong.tsg", "chordpro.csg", "subdir/chordpro.csg"]
```

Comment:
Ok. Du coup, un jour, les extensions .is, .json (ou .yml) et .tex seront reconnues et non pas ignorées, c'est ça ?
On a une issue pour ça, ou c'est seulement dans nos têtes ?